Xamarin.Forms.Platform.WPF/Renderers/BoxViewRenderer.cs

```diff
@@ -31,12 +27,21 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
 			if (e.PropertyName == BoxView.ColorProperty.PropertyName)
 				UpdateColor();
+			else if (e.PropertyName == BoxView.CornerRadiusProperty.PropertyName)
+				UpdateCornerRadius();
 		}
 
 		void UpdateColor()
 		{
 			Color color = Element.Color != Color.Default ? Element.Color : Element.BackgroundColor;
 			Control.UpdateDependencyColor(WRectangle.FillProperty, color);
 		}
+
+		void UpdateCornerRadius()
+		{
+			var cornerRadius = Element.CornerRadius;
+			Control.RadiusX = cornerRadius.TopLeft;
```

Comment:
Seems like this would ignore the values for the TopRight and BottomLeft corners, wouldn't it?

Reply:
You're right. I have changed it. Now every value is used. Previously a Rectangle was used. The quick and easy way to modify CornerRadius was with RadiusX and RadiusY. Now with a Border, the CornerRadius property is used.